ruby.c

```diff
@@ -271,6 +284,23 @@ usage(const char *name, int help)
 	SHOW(features[i]);
 }
 
+static void
+usage_dump(const char *name)
+{
+    static const struct message usage_msg[] = {
+	M("--dump=insns",     "", "dump instruction sequences"),
+	M("--dump=yydebug",   "", "dump yydebug of yacc parser generator"),
+	M("--dump=parsetree", ", --dump=parsetree_with_comment",
+          "dump AST or AST with comments"),
+    };
+    int i;
+    int help = 1; // used in SHOW
```

Comment:
You should stick with C90 style comments in our source tree: `/* ... */`

Reply:
fixed